_rules/image-button-non-empty-accessible-name-59796f.md

```diff
@@ -56,8 +56,12 @@ Each target element has an [accessible name][] that is neither empty (`""`), nor
 
 There is a known combination of a popular browser and assistive technology that does not by default support `title` as an [accessible name][].
 
+The [input type="image" Accessible Name Computation algorithm](https://www.w3.org/TR/html-aam/#input-type-image-accessible-name-computation) uses the first non-empty name, but some user agents and assistive technologies combinations stop at the first existing one, even if empty.
+
 ## Background
 
+Contrarily to images, an empty `alt` attribute does not make image button decorative; image buttons have a button role and are therefore exposed as interactive elements. Consequently, an empty `alt` attribute does not provide a "usable string" for image buttons and the computation defaults to other means of providing a name.
```

Comment:
1. I don't think you should use `Contrarily to images`, you're talking specifically about the `img` element. There are different types of images in HTML.
2. I'm not sure what that second sentence means. Can you make it clearer?